src/Adapter/Order/CommandHandler/UpdateProductInOrderHandler.php

```diff
@@ -34,6 +34,7 @@
 use Order;
 use OrderDetail;
 use OrderInvoice;
+use PrestaShop\Decimal\Number;
```

Comment:
```suggestion
use PrestaShop\Decimal\DecimalNumber;
```

Prefer DecimalNumber for PHP8

Reply:
How, it's maybe only working on the develop branch :D

Reply:
Not handled in 177, the new library is only available in 178